Examples/AblyPush/AblyPushExample/Buttons/PushButton.swift

```diff
@@ -0,0 +1,52 @@
+import SwiftUI
+
+struct PushButton: View {
+    let systemImage: String
+    let activeTitle: String
+    let inactiveTitle: String
+    let action: () -> Void
+    let isButtonEnabled: Bool // is the button enabled
+    @Binding var isActive: Bool // is the action in it's on or off state
+    
+    let activeColor = Color(red: 1.00, green: 0.33, blue: 0.09)
+    let inActiveColor = Color(red: 0.00, green: 0.56, blue: 0.02)
+    
+    init(
+        isButtonEnabled: Bool,
+        systemImage: String,
+        activeTitle: String,
+        inactiveTitle: String,
+        isActive: Binding<Bool>,
```

Comment:
Is binding needed here? `isButtonEnabled` works in similar way but without binding.

Reply:
It's required as it will only update the state of the button once the `Published` property has changed from the caller class. This ensures the button doesn't just toggle states without the action actually succeeding.

Reply:
Couldn't you elaborate? I've checked one more time and it works exactly the same without binding. I think we shouldn't confuse users with a sample code. If it works without this complication then it should be removed.

Reply:
Ahh my bad - I misunderstood what you were trying to say. I assumed you were for the removal of `isActive` as opposed to changing the type of it to just `Bool` instead of `Binding<Bool>` 😅 

All sorted in [240e927](https://github.com/ably/ably-cocoa/pull/1879/commits/240e927c223ed28d866cb273bfbf838560cecee2)